scripts/helper/functions.sh

```diff
@@ -35,7 +37,7 @@ verify_installed()
 preflight_checks()
 {
   # Verify appropriate tools are installed on host
-  for cmd in jq docker-compose keytool docker openssl xargs awk; do
+  for cmd in jq docker-compose keytool docker openssl xargs awk wget; do
```

Comment:
There's probably an existing discussion on this, but ... I assume the download could not be done in any reasonable way with `curl`?

Reply:
@javabrett wget --> curl, in latest commit.